src/jl_uv.c

```diff
@@ -1085,6 +1085,24 @@ JL_DLLEXPORT int jl_queue_work(work_cb_t work_func, void *work_args, void *work_
     return 0;
 }
 
+JL_DLLEXPORT int jl_uv_update_timer_start(uv_loop_t* loop, jl_value_t* jltimer,
+                                          uv_timer_t* uvtimer, uv_timer_cb cb,
+                                          uint64_t timeout, uint64_t repeat)
+{
+    JL_UV_LOCK();
+    int err = uv_timer_init(loop, uvtimer);
+    if (err) {
```

Comment:
this actually can't fail, so we can just delete the error checking code (or make it an abort)